pkg/ui/v1alpha3/frontend/src/sagas/index.js

```diff
@@ -12,18 +12,36 @@ export const submitYaml = function* () {
     while (true) {
         const action = yield take(generalActions.SUBMIT_YAML_REQUEST);
         try {
-            const result = yield call(
-                goSubmitYaml,
-                action.yaml
-            )
-            if (result.status === 200) {
-                yield put({
-                    type: generalActions.SUBMIT_YAML_SUCCESS,
-                })
+            let isRightNamespace = false
+            for (const [key, value] of Object.entries(action.yaml.split("\n"))) {
```

Comment:
Wouldn't the key, in this case, be an index? Since it's not an object, it may make sense to name the vars accordingly?

```suggestion
            for (const [index, line] of Object.entries(action.yaml.split("\n"))) {
```

Sorry if I misinterpreted this line

Reply:
Yes, it will be the index. I will change it to `[index, value]`